File: ios/Classes/Transfer/ExpressHandler.m
#import "ExpressHandler.h"

@implementation ExpressHandler

- (void) start : (UIViewController *) controller dictionary: (NSDictionary *) dictionary delegate: (id<P24TransferDelegate>) delegate {
      NSString* url = [dictionary objectForKey:@"url"];

      P24ExpressParams* params = [[P24ExpressParams alloc] initWithUrl: url];
      
      [P24 startExpress:params inViewController: controller delegate: delegate];
}

@end

File: ios/p24_sdk/Sources/p24_sdk/Transfer/ExpressHandler.swift
import P24

internal struct ExpressHandler : TransferHandler {
    func start(controller: UIViewController?, dictionary: [String: Any], delegate: P24TransferDelegate) {
        let params = P24ExpressParams(url: (dictionary["url"] as! String))!
        P24.startExpress(params, in: controller, delegate: delegate)
    }
}
